akka-http-circe/src/main/scala/de/heikoseeberger/akkahttpcirce/CirceSupport.scala

```diff
@@ -67,9 +72,7 @@ trait CirceSupport {
     */
   implicit def circeToEntityMarshaller[A](
       implicit encoder: Encoder[A],
-      printer: Json => String = Printer.noSpaces.pretty
+      printer: Json ⇒ String = Printer.noSpaces.pretty
   ): ToEntityMarshaller[A] =
-    Marshaller.StringMarshaller
-      .wrap(`application/json`)(printer)
-      .compose(encoder.apply)
+    jsonStringMarshaller.compose(printer).compose(encoder.apply)
```

Comment:
The behaviour is exactly the same.
The change hopefully makes the code more readable, but more importantly it should be more efficient (even if not benchmarked). 

The call to `Marshaller.StringMarshaller.wrap(``application/json``)` is simply factorized into a val to skip to compute over and over again the same value. The trade-off with the memory is worth it as it's one object in memory for all the application life.

I talked with Konrad Malawski regarding this on Gitter, he agreed that it will prevent some object allocation (and it's safe). The impact must be tiny though. Since it doesn't hurt the code clarity I included the change.